code/src/Summary.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export const Summary = ({ consumption, numberOfBooks, favouriteAuthor, onRestart}) => {
+    return (
+        <>
+            <section className="summary">
+                <p>Wow! You read {numberOfBooks} books each year!</p>
+                <p>It's awesome to {consumption}.</p>
+                <p>{favouriteAuthor} is a good choice.</p>
+            </section>
```

Comment:
some tab index here! The screen reader could not read this part

Reply:
I added a tabindex here (didn't do it before because I thought it was only supposed to be on interactive elements). Now it's tabable and the screen reader can read it. However, after clicking on the submit button, the reader hops over it and automatically goes on to the Restart-button. I have to tab backwards to access the summary. It seems to be a problem with the Submit-button, but I don't see where it's coming from.